### PR TITLE
AMP-58795 React Native V2 examples

### PR DESCRIPTION
**Pull request by falconandy**

### CODE REVIEW

File: react-native/javascript/v2/AmpliApp/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+2. `ampli pull`
+3. Update `App.js` to use the events from your tracking plan
+
+# Project structure
+* README.md - you are here *
+* src/
+    * [App.js](src/App.js) - Example user app using Amply SDK. A good place to start.
+    * ampli/
+        * [index.js](src/ampli/index.js) - Generated SDK, don't modify by hand. Update with `ampli pull`
+    * [middleware/*](src/middleware) - Example middlewares for Segment and Event filtering
```
Comment (justin-fiedler) [inline react-native/javascript/v2/AmpliApp/README.md:34]: *plugins

**Review by justin-fiedler — CHANGES_REQUESTED** [review]
Looking good. See nits.

File: react-native/javascript/v2/AmpliApp/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (falconandy) [reply react-native/javascript/v2/AmpliApp/README.md]: Fixed

File: react-native/typescript/v2/AmpliApp/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+2. `ampli pull`
+3. Update `App.tsx` to use the events from your tracking plan
+
+# Project structure
+* README.md - you are here *
+* src/
+    * [App.tsx](src/App.tsx) - Example user app using Amply SDK. A good place to start.
+    * ampli/
+        * [index.ts](src/ampli/index.ts) - Generated SDK, don't modify by hand. Update with `ampli pull`
+    * [middleware/*](src/middleware) - Example middlewares for Segment and Event filtering
```
[the remark this answers is not in the document]
Comment (falconandy) [reply react-native/typescript/v2/AmpliApp/README.md]: Fixed

File: react-native/javascript/v2/AmpliApp/README.md (hunk 1 above)
Comment (falconandy) [reply react-native/javascript/v2/AmpliApp/README.md]: Fixed

**Review by justin-fiedler — APPROVED** [review]

**Pull request merged**